specs/global/parameters.json

```diff
@@ -8,19 +8,19 @@
             "type": "string"
         }
     },
-    "entityId":{
+    "entity_id":{
```

Comment:
I'm not sure the actual property name has to be snake cased, this is really just an internal name and doesn't appear anywhere.

The only thing that needs to be changed is the value for the `name` key in this object. This will save us from all those changes below (and the potential mistakes they might entail)